app/templates/partials/ga.html

```diff
@@ -0,0 +1,14 @@
+<!-- Google tag (gtag.js) --> 
+<script nonce="{{ cspNonce }}" async src="https://www.googletagmanager.com/gtag/js?id={{ gtm_tag_id }}"></script> 
+
+<script nonce="{{ cspNonce }}">
+    var allowTrackingCookies = /^(.*)?\s*'campaign':true\s*[^;]+(.*)?$/;
```

Comment:
Any reason you used `campaign` instead of `usage` like previously? It should also be `campaigns` 
```suggestion
    var allowTrackingCookies = /^(.*)?\s*'campaigns':true\s*[^;]+(.*)?$/;
```